odoo/addons/base/tests/test_menu.py

```diff
@@ -28,3 +28,60 @@ def test_00_menu_deletion(self):
 
         orphans =  Menu.search([('id', 'in', all_ids), ('parent_id', '=', False)], order="id")
         self.assertEqual([child1.id, child2.id], orphans.ids)
+
+    def test_best_root_menu_for_model(self):
+        Menu = self.env['ir.ui.menu']
+        Action = self.env['ir.actions.act_window']
+        menu_cpt = 0
+
+        def new_menu(name, parent_id=None, action=None):
+            nonlocal menu_cpt
+            menu_cpt += 1
```

Comment:
You can use the Mock library instead of this counter (and use `call_count`)
https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.call_count

```suggestion
```

Reply:
Thanks for the tip. It is changed using the Mock library.